stuffer/s2n_stuffer_file.c

```diff
@@ -41,7 +41,7 @@ int s2n_stuffer_recv_from_fd(struct s2n_stuffer *stuffer, const int rfd, const u
     } while (r < 0);
 
     /* Record just how many bytes we have written */
-    POSIX_ENSURE(r <= UINT32_MAX, S2N_ERR_INTEGER_OVERFLOW);
+    POSIX_ENSURE((uint64_t) r <= UINT32_MAX, S2N_ERR_INTEGER_OVERFLOW);
```

Comment:
Would (size_t) make more sense here?

Same for the one in s2n_stuffer_send_to_fd